### DISCUSSION TITLE
L&R mapped merged only work on one controller at time

### DISCUSSION THREAD

**Discussion by rafaeoz** [darthcloud/BlueRetro#565] [Q&A] [answered]
I have the bluretro device 

![Screenshot_2023-01-18-21-18-35-015_com.alibaba.aliexpresshd.jpg](https://user-images.githubusercontent.com/114241445/213326148-886c2f72-b403-4e8d-bb32-0813b5722f0a.jpg)

And when i map in bluretro.io only one controller is set merged...i tried map all 4 imputs. W

**Comment by darthcloud** [comment, marked as answer]
Use the discussion forum for question, not issues.

You need to set the preset for each output.

**Reply by rafaeoz** [reply]
Ok. Sorry. 
I set for each output. 1 to 4. i even tried to set all the 12. I noticed that only the last one that i set work.

**Reply by rafaeoz** [reply]
> Use the discussion forum for question, not issues.
> 
> You need to set the preset for each output.

I noticed now...this happen when i sync 2 or more  dualshock 4. If i set one dualshock 4 and 1 dualshock 3 work on both.
.
When i sync only Ds4 ...only one (usually the last one mapped) work.

**Comment by darthcloud** [comment]
I saved the merge trigger preset on the first 4 output.

Then I connected an xbox one, PS5, PowerA GC and PS4 controller and the trigger on all four worked as expected.

All 4 result in both the analog and digital section being presses for each L & R trigger

So not sure why it doesn't work for you.

**Reply by rafaeoz** [reply]
You tried 2 ds4 or more at the same time?
Because when i use a ds4 and a ds3...they work ok with the merged
